uint/CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+
+## [0.9.2] - 2022-01-28
 - Migrated to 2021 edition, enforcing MSRV of `1.56.1`. [#601](https://github.com/paritytech/parity-common/pull/601)
```

Comment:
shouldn't the MSRV bump be regarded as a breaking change?

Reply:
see my comment above

Reply:
fair enough, got it